### PR TITLE
Issue #1250: Use consistent terminology for URL alias.

### PR DESCRIPTION
**Pull request by jenlampton**
PR for https://github.com/backdrop/backdrop-issues/issues/1250

### CODE REVIEW

File: core/modules/path/path.path.inc [hunk 1]
```diff
@@ -175,8 +175,8 @@ function taxonomy_path_bulk_update_batch_process(&$context) {
 function user_path_info() {
   $info['user'] = array(
     'entity type' => 'user',
-    'label' => t('User paths'),
-    'pattern description' => t('Pattern for user account page paths'),
+    'label' => t('User Account'),
```
Comment (quicksketch) [inline core/modules/path/path.path.inc]: "Account" should be lower case. As with nodes and terms, a replacement word seems necessary here.

File: core/modules/path/tests/path_pattern.test [hunk 2]
```diff
@@ -794,11 +794,11 @@ class PathPatternBulkUpdateTestCase extends PathPatternFunctionalTestHelper {
...
     $edit = array(
-      'update[node_path_bulk_update_batch_process]' => TRUE,
-      'update[taxonomy_path_bulk_update_batch_process]' => TRUE,
-      'update[user_path_bulk_update_batch_process]' => TRUE,
+      'generate[update][node_path_bulk_update_batch_process]' => TRUE,
+      'generate[update][taxonomy_path_bulk_update_batch_process]' => TRUE,
+      'generate[update][user_path_bulk_update_batch_process]' => TRUE,
     );
-    $this->backdropPost('admin/config/search/path/bulk-update', $edit, t('Bulk generate paths'));
+    $this->backdropPost('admin/config/search/path/bulk-update', $edit, t('Generate URL Aliases'));
```
Comment (quicksketch) [inline core/modules/path/tests/path_pattern.test]: By convention we use sentence casing, so "Aliases" should be lower case.

File: core/modules/path/tests/path_pattern.test [hunk 3]
```diff
@@ -811,7 +811,7 @@ class PathPatternBulkUpdateTestCase extends PathPatternFunctionalTestHelper {
     $new_node = $this->backdropCreateNode(array('path' => array('alias' => '', 'auto' => FALSE)));
 
     // Run the update again which should only run against the new node.
-    $this->backdropPost('admin/config/search/path/bulk-update', $edit, t('Bulk generate paths'));
+    $this->backdropPost('admin/config/search/path/bulk-update', $edit, t('Generate URL Aliases'));
```
Comment (quicksketch) [inline core/modules/path/tests/path_pattern.test]: "Aliases" lower case.

File: core/modules/path/path.path.inc [hunk 4]
```diff
@@ -14,8 +14,8 @@
 function node_path_info() {
   $info['node'] = array(
     'entity type' => 'node',
-    'label' => t('Content paths'),
-    'pattern description' => t('Default path pattern (applies to all content types with blank patterns below)'),
+    'label' => t('Content'),
```
Comment (quicksketch) [inline core/modules/path/path.path.inc:17]: "Content" is describing "paths" here. If we remove "paths" it should be replaced with another word of some sort. "URLs" or "patterns".

File: core/modules/path/path.path.inc [hunk 5]
```diff
@@ -101,8 +101,8 @@ function node_path_bulk_update_batch_process(&$context) {
 function taxonomy_path_info() {
   $info['taxonomy_term'] = array(
     'entity type' => 'taxonomy_term',
-    'label' => t('Taxonomy term paths'),
-    'pattern description' => t('Default path pattern (applies to all vocabularies with blank patterns below)'),
+    'label' => t('Taxonomy term'),
```
Comment (quicksketch) [inline core/modules/path/path.path.inc:104]: Same as "Content" above, we need a replacement for the word "paths" here.

File: core/modules/path/path.admin.inc [hunk 6]
```diff
@@ -19,8 +19,8 @@ function path_admin_overview($keys = NULL) {
   $multilanguage = (module_exists('language') || $alias_exists);
 
   $header = array();
-  $header[] = array('data' => t('Alias'), 'field' => 'alias', 'sort' => 'asc');
-  $header[] = array('data' => t('System'), 'field' => 'source');
+  $header[] = array('data' => t('URL Alias'), 'field' => 'alias', 'sort' => 'asc');
```
Comment (quicksketch) [inline core/modules/path/path.admin.inc]: "Alias" should be lower case.

File: core/modules/path/path.admin.inc [hunk 7]
```diff
@@ -126,17 +126,17 @@ function path_admin_form($form, &$form_state, $path = array('source' => '', 'ali
     '#default_value' => $path['source'],
     '#maxlength' => 255,
     '#size' => 45,
-    '#description' => t('Specify the existing path you wish to alias. For example: node/28, taxonomy/term/1.'),
+    '#description' => t('Specify the existing path you wish to alias. For example: node/28, or taxonomy/term/1.'),
```
Comment (quicksketch) [inline core/modules/path/path.admin.inc]: The comma can be removed here. Should be `node/28 or taxonomy/term/1`.

File: core/modules/path/path.path.inc (hunk 4 above)
Comment (jenlampton) [inline core/modules/path/path.path.inc:17]: The problem is that the noun changes depending on which admin page we are on. For example, in some cases the noun is "URL aliases" and in some places the noun is "URL patterns". See

``` php
'#title' => t('!label URL aliases', array('!label' => $path_info['label'])),
```

``` php
$group_header = t('!label URL patterns', array('!label' => $path_info['label']));
```

File: core/modules/path/path.path.inc (hunk 5 above)
Comment (jenlampton) [inline core/modules/path/path.path.inc:104]: Same explanation as above :)

**Comment by quicksketch** [comment]
Excellent. Merged into 1.x for 1.3.0.

**Pull request merged**